src/pods/test-sentences/test-sentences.component.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import * as styles from 'common/styles/tests.styles';
+import ButtonGroup from '@material-ui/core/ButtonGroup';
+import { Typography, Button } from '@material-ui/core';
+import { SentenceEntityApi } from 'core/sentences';
+import { VerbEntityGlobal } from 'core/verbs';
+
+interface Props {
+  sentenceSelected: SentenceEntityApi;
+  selectedVerb: VerbEntityGlobal;
+  splitSentence: (sentence: string) => string[];
+}
+
+export const TestSentencesComponent: React.FC<Props> = props => {
+  const { sentenceSelected, selectedVerb, splitSentence } = props;
+  const [firstSentencePart, secondSentencePart] = splitSentence(
+    sentenceSelected.sentence
+  );
```

Comment:
You can create a custom hook for this, let's sit down this weekend (if you have time and we take a look)

Reply:
This will be assumed by the mapper